test/functional/feature_dirsymlinks.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# Copyright (c) 2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test successful startup with symlinked directories.
+"""
+
+import os
+import sys
+
+from test_framework.test_framework import BitcoinTestFramework, SkipTest
+
+
+def rename_and_link(fromname, toname):
+    os.rename(fromname, toname)
+    os.symlink(toname, fromname)
```

Comment:
pico nit if need to retouch, snakecase `from_name, to_name` and use named arguments in the callers (feel free to ignore)

Reply:
Yea, that'd be slightly better.